packages/babel-plugin-transform-function-name/package.json

```diff
@@ -18,7 +18,8 @@
   ],
   "dependencies": {
     "@babel/helper-function-name": "workspace:^",
-    "@babel/helper-plugin-utils": "workspace:^"
+    "@babel/helper-plugin-utils": "workspace:^",
+    "@babel/helper-compilation-targets": "workspace:^"
```

Comment:
Please run `yarn install` when new dependencies are added and commit the changed yarn lock file.

Reply:
Okay, I will run `yarn install` and commit the changes in yarn lock file.